CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ listed in the changelog.
 
 - Stream Buildah and Aqua log output ([#596](https://github.com/opendevstack/ods-pipeline/issues/596))
 - Update skopeo (1.6 to 1.8) and buildah (1.24 to 1.26) ([#598](https://github.com/opendevstack/ods-pipeline/issues/598))
+- Support running different pipelines on different webhook events ([#562](https://github.com/opendevstack/ods-pipeline/issues/562))
```

Comment:
I saw you added support for the "legacy" format - something to mention here? How long would we support the old format? Until 0.8?

Reply:
Will mention that in the changelog entry. Not sure if we should deprecate the old format at all - some initiatives might not want to specify trigger conditions at all and would be fine with what we used so far?